templates/os_configure/extend_to_bash_aliases

```diff
@@ -0,0 +1,14 @@
+
+## open bash specific files
+alias 'open_bash'='vim ~/.bashrc'
+alias 'open_commands'='vim ~/.bash_commands'
+alias 'open_aliases'='vim ~/.bash_aliases'
+alias 'open_rtw_file'='vim ~/.ros_team_ws_rc'
+
+## sourcing
+alias 'source_bash'='source ~/.bashrc'
+#ros
+alias 'source_foxy'='source /opt/ros/foxy/setup.bash'
+alias 'source_galactic'='source /opt/ros/galactic/setup.bash'
+alias 'source_humble'='source /opt/ros/humble/setup.bash'
+alias 'source_rolling'='source /opt/ros/rolling/setup.bash'
```

Comment:
I am not so sure about those aliases. They are not really shorting the commands but add potential confusion.

The source part is also not so often used when one is having ros_team_workspace installed. Maybe should have rather there a functionality then adding this alias.

Reply:
Should i delete them?

Reply:
This should be part of the RTW

Reply:
only have alias for 'source ~/.bashrc', kate instead of vim, distro sourcing: should we check if the path exists?